lib/api/src/tabs/events.ts

```diff
@@ -26,7 +26,14 @@ export class TabCreatedEvent extends TabEvent {
  * most common is loading new url.
  */
 export class TabUpdatedEvent extends TabEvent {
-  //
+
+  /**
+   * @param tabId Id of tab that was activated
+   */
+  constructor (tabId: number, public changeInfo: chrome.tabs.TabChangeInfo) {
```

Comment:
Quite like this, plus the `chrome.tabs.` prefix clearly indicates that this interface is not own my Exteranto.

Reply:
I think that at the very least we need to export exteranto interface that extends the chrome one (in which case we might as well add the 4 properties manually), because exteranto does not export chrome types and the type would not be accessible in the app.